python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -446,6 +447,36 @@ def config_type_run_launcher(cls):
                     description="Raw Kubernetes configuration for launched runs.",
                 ),
                 "job_namespace": Field(StringSource, is_required=False, default_value="default"),
+                "only_allow_user_defined_k8s_config_fields": Field(
+                    Shape(
+                        {
+                            "container_config": Field(
+                                Map(key_type=str, inner_type=bool), is_required=False
+                            ),
+                            "pod_spec_config": Field(
+                                Map(key_type=str, inner_type=bool), is_required=False
+                            ),
+                            "job_metadata": Field(
+                                Map(key_type=str, inner_type=bool), is_required=False
+                            ),
+                            "job_spec_config": Field(
+                                Map(key_type=str, inner_type=bool), is_required=False
+                            ),
+                            "namespace": Field(bool, is_required=False),
```

Comment:
do you think it would be clearer if this were "allowed_container_config" / "allowed_pod_spec_config" etc?